### ISSUE TITLE
Camera preview is upside down

### ISSUE THREAD

**Issue by mobiledev1221** [muxinc/examples#53]
The camera preview is upside down when I opened this demo application. Could you let me know how I can correct it?
![Screenshot_20201220-063447](https://user-images.githubusercontent.com/50055908/102701026-9fe94000-4207-11eb-907b-d022f07b253f.png)

**Comment by GeneticGenesis** [comment]
Hey @mobiledev1221, this is actually a known issue in the library that powers this example, as described in the readme.

https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/issues/266

What device and OS version are you seeing this on?

Thanks.

**Comment by GeneticGenesis** [comment]
Hey @mobiledev1221, were you able to confirm which device this was on?

Thanks!

**Comment by mobiledev1221** [comment]
I was on nexus5x.

Thank you

**Comment by GeneticGenesis** [comment]
Thanks, unfortunately this is expected on the Nexus 5x and 6p right now as per https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/issues/266

Unfortunately the Nexus 5X was released in 2013, and stopped receiving updates in 2018. This isn't something we can currently put a lot of time into fixing bugs for unfortunately, especially when they're in the underlying library.

I suggest you comment on the ticket I've linked adding the 5x to the list of devices with this issue, and hopefully the maintainers of rtmp-rtsp-stream-client-java will take a look.

Thanks!
